source/iterators.tex

```diff
@@ -4043,14 +4051,32 @@
 
 \indexlibrarymember{base}{move_iterator}%
 \begin{itemdecl}
-constexpr Iterator base() const;
+constexpr Iterator base() const &;
 \end{itemdecl}
 
 \begin{itemdescr}
+\pnum
+\constraints \tcode{Iterator} satisfies
+the \oldconcept{CopyConstructible} requirements.
+
+\pnum
+\effects \tcode{Iterator} models
+the \oldconcept{CopyConstructible} requirements.
```

Comment:
Ditto `models \libconcept{CopyConstructible}.`

Reply:
Oh, and `\effects` should be `\expects` here.

Reply:
thanks!